courses/free-courses-hi.md

```diff
@@ -127,6 +127,7 @@
 * [C++ STL \| Competitive Programming](https://youtube.com/playlist?list=PLauivoElc3gh3RCiQA82MDI-gJfXQQVnn) - Luv
 * [C++ Tutorial For Begineers In Hindi](https://www.youtube.com/playlist?list=PLnSDvcENZlwA6YDSfoieM1bl-Y3ALcnL5) - Abhishek Shrivastava, Micro Solution
 * [C++ Tutorial for Beginners \| C++ Tutorials In Hindi](https://www.youtube.com/playlist?list=PLmGElG-9wxc8VMy1nNHDQldH2dU8Y08s7) - Manish Gehlot, WsCube Tech Programming Concepts
+* [C++ Tutorial for Beginners \| Full Course](https://www.youtube.com/watch?v=vLnPwxZdW4Y) - FreeCodeCamp
```

Comment:
This file is only for Hindi language Courses. Please move this link to English file

Reply:
Thanks for the review @shrn01 
@ec-026 please change accordingly

Reply:
Ok. I have changed it now. Please refer to it.